### DISCUSSION TITLE
Do you need a proxy for adblockers if you are using the npm package?

### DISCUSSION THREAD

**Discussion by maximedupre** [plausible/analytics#2102] [Anything Else]
Am I correct to think that using the npm package + bundling it with the rest of your code will circumvent ad blockers?

So using a proxy is only for those using the script tag?

Thanks :)

**Comment by metmarkosaric** [comment]
hi @maximedupre! yes, that's correct. we also list the npm package as an alternative to the proxy in our docs

**Comment by sandstrom** [comment]
I'd say blockers will also block the reporting back to plausible, so you need to proxy them too.

Unfortunately that path is hard-coded right now, so it's easy to block regardless + may collide with paths on your domain.

There is an open discussion about this here:
https://github.com/plausible/plausible-tracker/issues/32#issuecomment-1299965622

And also this underlying problem which you can vote for:
https://github.com/plausible/analytics/discussions/2414
